src/linear_algebra/matrix/schur_complement.lean

```diff
@@ -215,6 +216,153 @@ end
 
 end triangular
 
+/-! ### 2×2 block matrices -/
+
+section block
+
+/-! #### General 2×2 block matrices-/
+
+/-- A block matrix is invertible if the bottom right corner and the corresponding schur complement
+is. -/
```

Comment:
Somewhere in this file you can add a cross-reference (and I guess in the other direction too!) to `category_theory.preadditive.biproducts`, which proves very similar results. I guess biprod.guassian and biprod.iso_elim are the most directly relevant ones.

Reply:
(That is in slightly more generality as it is about 2x2 matrices in a category rather than a ring, but whatever. That file also only does one direction of these invertibility results, and doesn't (I think) give the iff statements.)